README.md

```diff
@@ -1,5 +1,5 @@
 <p align="center">
-  <img src="docs/_static/logo.png" width="200" alt="django-jsonform icon">
+  <img src="https://github.com/bhch/django-jsonform/raw/master/docs/_static/logo.png" width="200" alt="django-jsonform icon">
```

Comment:
This link returns a 302 redirect to the image's actual location at:

```
https://raw.githubusercontent.com/bhch/django-jsonform/master/docs/_static/logo.png
```

It seems better to just use the direct link instead.

Reply:
Why wouldn't they follow redirects? It's a part of HTTP standard.

Besides, Pypi is only one example. The relative link works only on github. There are other places where you'll display the readme file, like inside IDE. I tested in Pycharm, the absolute url works when readme is rendered with Markdown plugin.

Reply:
> The relative link works only on github.

I understand that. But I meant to use the direct link to avoid redirect: 

```suggestion
  <img src="https://raw.githubusercontent.com/bhch/django-jsonform/master/docs/_static/logo.png" width="200" alt="django-jsonform icon">
```